[2023-05-24T15:15:11.000+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2023-05-24T15:15:11.127+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2023-05-24T15:15:11.127+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2023-05-24T15:15:11.445+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2021-10-31 00:00:00+00:00
[2023-05-24T15:15:11.466+0000] {standard_task_runner.py:57} INFO - Started process 208 to run task
[2023-05-24T15:15:11.537+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_name', 'scheduled__2021-10-31T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpbmg6send']
[2023-05-24T15:15:11.601+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask get_name
[2023-05-24T15:15:12.749+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2021-10-31T00:00:00+00:00 [running]> on host 942de29f183d
[2023-05-24T15:15:14.567+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Sai' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2021-10-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-10-31T00:00:00+00:00'
[2023-05-24T15:15:14.569+0000] {python.py:183} INFO - Done. Returned value was: {'first_name': 'Sai', 'last_name': 'Tr'}
[2023-05-24T15:15:14.972+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=get_name, execution_date=20211031T000000, start_date=20230524T151511, end_date=20230524T151514
[2023-05-24T15:15:15.854+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-05-24T15:15:16.074+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-05-25T06:49:01.792+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2023-05-25T06:49:01.848+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2023-05-25T06:49:01.848+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2023-05-25T06:49:02.018+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2021-10-31 00:00:00+00:00
[2023-05-25T06:49:02.028+0000] {standard_task_runner.py:57} INFO - Started process 1749 to run task
[2023-05-25T06:49:02.053+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_name', 'scheduled__2021-10-31T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpxu7tcxnc']
[2023-05-25T06:49:02.057+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask get_name
[2023-05-25T06:49:02.449+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2021-10-31T00:00:00+00:00 [running]> on host 6bd07c7477d9
[2023-05-25T06:49:03.968+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Sai' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2021-10-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-10-31T00:00:00+00:00'
[2023-05-25T06:49:03.973+0000] {python.py:183} INFO - Done. Returned value was: {'first_name': 'Sai', 'last_name': 'Tr'}
[2023-05-25T06:49:04.232+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=get_name, execution_date=20211031T000000, start_date=20230525T064901, end_date=20230525T064904
[2023-05-25T06:49:04.321+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-05-25T06:49:04.626+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-25T10:16:32.431+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2023-05-25T10:16:33.125+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2021-10-31T00:00:00+00:00 [queued]>
[2023-05-25T10:16:33.130+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2023-05-25T10:17:19.695+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2021-10-31 00:00:00+00:00
[2023-05-25T10:17:20.068+0000] {standard_task_runner.py:57} INFO - Started process 1281 to run task
[2023-05-25T10:17:20.257+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_name', 'scheduled__2021-10-31T00:00:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp2psxa4_f']
[2023-05-25T10:17:20.322+0000] {standard_task_runner.py:85} INFO - Job 18: Subtask get_name
[2023-05-25T10:18:13.131+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 18 for task get_name ((psycopg2.OperationalError) could not translate host name "postgres" to address: Temporary failure in name resolution

(Background on this error at: https://sqlalche.me/e/14/e3q8); 1281)
[2023-05-25T10:18:15.651+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-05-25T10:18:27.117+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
